src/client/nativeCommunication.ts

```diff
@@ -56,6 +60,7 @@ function insertAds(): void {
             }
         };
 
+        window.addEventListener('orientationchange', callback);
```

Comment:
don't love this but that's the only way I managed to get the ads to update on orientation change .. also this won't work until I make a PR iOS side as right now the update adverts code in iOS only updates the `y` position 🤷‍♂️

Reply:
https://github.com/guardian/ios-live/pull/5218

Reply:
Would this still work?

We'll need to update ad positions when users multi-task multiple apps https://support.apple.com/en-gb/HT207582

```suggestion
        window.addEventListener('resize', callback);
```

Reply:
This actually even works better as it repositions the ad immediately instead of after the rotation animation finishes 👍